Winget-AutoUpdate/functions/Update-App.ps1

```diff
@@ -18,36 +18,36 @@ Function Update-App ($app) {
     
     #Run Winget Upgrade command
     & $Winget upgrade --id $($app.Id) --all --accept-package-agreements --accept-source-agreements -h | Tee-Object -file $LogFile -Append
-    
+
+    #Check if mods exist for upgrade
+    $ModsInstall, $ModsUpgrade = Test-Mods $($app.Id)
+    if ($ModsUpgrade) {
+        Write-Log "Modifications for $($app.Id) after upgrade are being applied..." "Yellow"
+        & "$ModsUpgrade"
+    }
+
     #Check if application updated properly
     $CheckOutdated = Get-WingetOutdatedApps
     $FailedToUpgrade = $false
-    #Check if mods exist
-    $ModsInstall, $ModsUpgrade = Test-Mods $($app.Id)
     foreach ($CheckApp in $CheckOutdated) {
         if ($($CheckApp.Id) -eq $($app.Id)) {
             
             #If app failed to upgrade, run Install command
             & $Winget install --id $($app.Id) --accept-package-agreements --accept-source-agreements -h | Tee-Object -file $LogFile -Append
-            
+
+            #Check if mods exist for install
+            $ModsInstall, $ModsUpgrade = Test-Mods $($app.Id)
```

Comment:
This action is already realized line 23 no ?

Reply:
No line 23 is the first loop when upgrading, line 39 is when the upgrade has failed and it tries the install command

Reply:
But yes. You'r right. It's already checked so...